test/cmd/release-notification-listener/listener/tekton.go

```diff
@@ -0,0 +1,118 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package listener
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+)
+
+const (
+	gitSHAMaxLength = 7
+	noPrNumber      = "none"
+)
+
+var (
+	tektonCLICommandPath string
+	pipelinesAndFilters  = map[string][]string{
+		"suite": []string{
+			"Integration",
+			"Consolidation",
+			"Utilization",
+			"Interruption",
+			"Chaos",
+			"Drift",
+		},
+		"ipv6": []string{},
+	}
+)
+
+func runTests(message *notificationMessage, args ...string) error {
+	log.Printf("running tests with args %v for release type: %s, release identifier: %s ", args, message.ReleaseType, message.ReleaseIdentifier)
+	cmd := exec.Command(tektonCLICommandPath, args...)
```

Comment:
It strikes me that using the kubernetes go client and the tekton apis is probably easier/cleaner. Curious why you went this route -- if you're not familiar with k8s client-go, it's worth the investment to learn it.  Happy to merge as is.

Reply:
I did attempt that there are some obstacles. Should be able to replace these parts when it's ready.